spring-restdocs-mockmvc/src/test/java/org/springframework/restdocs/mockmvc/MockMvcRestDocumentationIntegrationTests.java

```diff
@@ -157,8 +159,24 @@ public void curlSnippetWithContent() throws Exception {
 				new File(
 						"build/generated-snippets/curl-snippet-with-content/curl-request.adoc"),
 				is(snippet(asciidoctor()).withContents(codeBlock(asciidoctor(), "bash")
-						.content("$ curl " + "'http://localhost:8080/' -i -X POST "
-								+ "-H 'Accept: application/json' -d 'content'"))));
+						.content(String.format("$ curl " + "'http://localhost:8080/' -i -X POST "
+								+ "\\%n -H 'Accept: application/json' \\%n -d 'content'")))));
+	}
+
+	@Test
+	public void curlSnippetWithContentSpaceSeparator() throws Exception {
```

Comment:
It's good to have tests for the curl snippet with different command formatters, but there's nothing MockMvc-specific about it so I don't think they belong here.